rundeckapp/grails-app/init/rundeckapp/Application.groovy

```diff
@@ -86,8 +86,7 @@ class Application extends GrailsAutoConfiguration implements EnvironmentAware {
     }
 
     void loadGroovyRundeckConfigIfExists(final Environment environment) {
-        String rundeckGroovyConfigFile = System.getProperty(RundeckInitConfig.SYS_PROP_RUNDECK_SERVER_CONFIG_DIR) +
-                                         "/rundeck-config.groovy"
+        String rundeckGroovyConfigFile = System.getProperty(RundeckInitConfig.SYS_PROP_RUNDECK_CONFIG_LOCATION)
```

Comment:
Your code should check if there is a file provided by RundeckInitConfig.SYS_PROP_RUNDECK_CONFIG_LOCATION, and if not, it needs to check it there is a rundeck-config.groovy in the server config directory, which is what the previous code would load.

Reply:
https://github.com/danifr/rundeck/blob/28f838350d228f9750aff8048ff366e04e18c65f/rundeckapp/grails-app/init/rundeckapp/Application.groovy#L92